### PR TITLE
LDP 6757 First pr

### PR DESCRIPTION
**Pull request by alvarosaugarlr**
First pr of [LPD-6757](https://liferay.atlassian.net/browse/LPD-6757)

The changes requested in [this comment](https://github.com/brianchandotcom/liferay-portal/pull/154657#issuecomment-2381601826) have been made.

### CODE REVIEW

File: modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/portlet/action/SaveScimConfigurationMVCActionCommand.java [hunk 1]
```diff
@@ -128,13 +128,13 @@ protected void doProcessAction(
 				Date accessTokenExpirationDate =
 					oAuth2Authorization.getAccessTokenExpirationDate();
 
-				int daysBetween = DateUtil.getDaysBetween(
+				int daysToExpireAccessToken = DateUtil.getDaysBetween(
```
Comment (stian-sigvartsen) [inline modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/portlet/action/SaveScimConfigurationMVCActionCommand.java:131]: Are you sure the old old name is not following SF conventions more?
I think convention suggests it should either be called "daysBetween" or simply "date" (named after the type.

**Comment by stian-sigvartsen** [comment]
ci:test:sf

**Comment by daiazius** [comment]
Hi, I just pushed a commit that should fix the playwright test in CI

**Comment by alvarosaugarlr** [comment]
ci:test:playwright

**Comment by alvarosaugarlr** [comment]
ci:test:sf

**Comment by alvarosaugarlr** [comment]
ci:test:playwright

**Comment by alvarosaugarlr** [comment]
ci:test:stable

**Comment by alvarosaugarlr** [comment]
ci:test:security

**Comment by daiazius** [comment]
All SCIM's Playwright tests are passing, so all good

> ### ❌ ci:test:playwright - 54 out of 80 jobs passed in 2 hours 34 minutes
> Click here for more details.

File: modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/portlet/action/SaveScimConfigurationMVCActionCommand.java (hunk 1 above)
Comment (alvarosaugarlr) [reply modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/portlet/action/SaveScimConfigurationMVCActionCommand.java]: Updated https://github.com/liferay-appsec/liferay-portal/pull/1977/commits/20c84a12b456ff0fc4cbe1a03e058e207ef00790

**Comment by alvarosaugarlr** [comment]
ci:forward:force

**Pull request closed without merging**